scripts/seed-supabase-test-user.js

```
@@ -0,0 +1,149 @@
+// Seed Supabase test user and data for integration tests
+// Usage: node scripts/seed-supabase-test-user.js
+
+require('dotenv').config();
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)